src/component/control/checkbox/index.js

```diff
@@ -1,28 +1,21 @@
-import { state } from '../../state';
-import { data } from '../../data';
-import { bookmark } from '../../bookmark';
 
 import * as form from '../../form';
 
-import { Button } from '../../button';
-import { Collapse } from '../../collapse';
 
-import { node } from '../../../utility/node';
 import { get } from '../../../utility/get';
 import { set } from '../../../utility/set';
-import { convertColor } from '../../../utility/convertColor';
-import { isValidString } from '../../../utility/isValidString';
 
-export const Control_checkbox = function({
+export const Control_checkbox = function ({
   object = {},
   id = 'name',
   path = false,
   labelText = 'name',
   description = false,
   action = false,
-  inputButton = false,
-  inputHide = false,
-  inputButtonStyle = false
+  //FIXME deprecated properties
+  inputButton = false, //eslint-disable-line
+  inputHide = false, //eslint-disable-line
+  inputButtonStyle = false //eslint-disable-line
```

Comment:
lines the properties on 16, 17, 18 are indeed used in a few places. Why would eslint flag these?

Reply:
They are not used in the function.

Reply:
I see, I'll do more digging into this outside this PR.